cluster-install/install.sh

```diff
@@ -1627,41 +1626,18 @@ step_check_kubeconfig_choice() {
 
 # step_query_user_for_resource_count is used to query the user on the number of nodes/namespaces they expect
 # to have on their cluster, which will help us recommend the appropriate resource limit preset.
-step_query_user_for_resource_count() {
```

Comment:
After talking with @nhammern, we agreed to reduce some of the prompting for resource limits since there was quite a bit.  This will remove the confirmation for node/namespace detection, instead going directly to the recommendation. Might further reduce them later on, but for now this is probably fine.